lib/Archive/Tar/Stream.pm

```diff
@@ -525,7 +576,7 @@ sub ParseHeader {
     s/\0.*//; # strip from first null
   }
 
-  my $chksum = oct($items[6]);
```

Comment:
You can use `eval { oct($items[6]) };` instead, which still gets compiled properly.